src/components/buttonME.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const buttonME = () => <span> test</span>;
```

Comment:
Component names must start with a Capital letter.  (If you don't, React will think you are using an html tag. For example, `<buttonME/>` will not render correctly.)